src/search/landmarks/landmark_graph.h

```diff
@@ -78,9 +78,10 @@ class LandmarkGraph {
 
     utils::HashMap<FactPair, LandmarkNode *> simple_landmarks_to_nodes;
     utils::HashMap<FactPair, LandmarkNode *> disjunctive_landmarks_to_nodes;
+    utils::HashMap<FactPair, std::vector<LandmarkNode *>> conjunctive_landmarks_to_nodes;
     Nodes nodes;
 
-    void remove_node_occurrences(LandmarkNode *node);
+    void remove_node_occurrences(LandmarkNode *node, bool erase_conjunctive);
```

Comment:
Also here, I'm assuming that the options erase_conjunctive and store_conjunctive are removed and the code behaves as in the true case.